### PR TITLE
Clarify multi-store/site config process

### PR DESCRIPTION
**Pull request by ccondra**
## Purpose of this pull request

This pull request (PR) links the user guide to related configuration topics in DevDocs for multi-store and multi-website installations, and adds an additional step at the end of instructions to configure the server environment.  (See GitHub issue #968) .

## Magento release version
 
- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [ ] Magento Open Source only
- [ ] Magento Commerce only

Is this update specific to an installed feature extension

- [ ] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

- [Website, Stores, and Views](https://docs.magento.com/user-guide/stores/websites-stores-views.html)
- [Adding Stores](https://docs.magento.com/user-guide/stores/stores-all-create-store.html)
- [Adding Websites](https://docs.magento.com/user-guide/stores/stores-all-create-website.html)

## Additional information

n/a

### CODE REVIEW

File: src/stores/stores-all-create-store.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ A single installation of Magento can have multiple stores that share the same Ad
 
 The important thing to understand is that the stores use the same Magento code and share the same Admin. Each store can have a separate catalog, or the stores can share the same catalog. Each store can have a separate [root category]({% link catalog/category-root.md %}), which makes it possible to have a different main menu for each store. Stores can also have different branding, presentation, and content. Take some time to plan your store hierarchy with future growth in mind before you begin, because it is used throughout the configuration.
 
+Multi-store installations of Magento must be configured from the Admin and also from the command line of the server.  The Magento developer [documentation](https://devdocs.magento.com/guides/v2.3/config-guide/multi-site/ms_over.html) provides detailed instructions for configuring the server environment.
```
Comment (shrielenee) [inline src/stores/stores-all-create-store.md:9]:
```suggestion
Multi-store installations of Magento must be configured from the Admin and also from the command line of the server. The Magento developer [documentation](https://devdocs.magento.com/guides/v2.3/config-guide/multi-site/ms_over.html) provides detailed instructions for configuring the server environment.
```

**Review by shrielenee — APPROVED** [review]
One minor multiple space removal, but approved.

**Comment by ccondra** [comment]
running tests

**Comment by ccondra** [comment]
running tests

**Pull request merged**